tests/functional/File_Logging_Test.php

```diff
@@ -0,0 +1,174 @@
+<?php
+namespace Tribe\Common;
+
+use Codeception\TestCase\WPTestCase,
+    Tribe__Log,
+    Tribe__Log__Logger,
+    Tribe__Main;
+
+class Logging_Test extends WPTestCase {
```

Comment:
If we switch, definitely - good call! Let's hash out that decision (about switching to the FS API) some more first.